library/alloc/src/string.rs

```diff
@@ -1678,10 +1679,14 @@ impl String {
         drop(guard);
     }
 
-    /// Inserts a character into this `String` at a byte position.
+    /// Inserts a character into this `String` at byte position `idx`.
+    ///
+    /// Reallocates if `self.capacity()` is insufficient,
+    /// which may involve copying all `self.capacity()` bytes.
+    /// Makes space for the insertion,
+    /// by copying all bytes of `&self[idx..]` to new positions.
```

Comment:
A comma isn't needed before "by" - it's not a comma splice per se, but it's a short sentence and not really someplace you would pause speaking it out loud.